motoman_sia20d_descartes/test/motoman_sia20d_descartes_plugin.cpp

```diff
@@ -0,0 +1,51 @@
+#include <ros/ros.h>
+#include <descartes_core/robot_model.h>
+#include <pluginlib/class_loader.h>
+#include <gtest/gtest.h>
+
+static const std::string PLUGIN_NAME = "motoman_sia20d_descartes/MotomanSia20dRobotModel";
+static const std::string GROUP_NAME = "manipulator";
+static const std::string WORLD_FRAME = "world_frame";
+static const std::string TCP_FRAME = "tool0";
+static const std::string ROBOT_DESCRIPTION = "robot_description";
+Eigen::Affine3d tcp_pose;
+
+pluginlib::ClassLoader<descartes_core::RobotModel> robot_model_loader("descartes_core",
+                                                                     "descartes_core::RobotModel");
+descartes_core::RobotModelPtr robot_model_ptr;
+
+TEST(MotomanSia20dRobotModel, createPluginInstance)
```

Comment:
This test could be generic for all ikfast solutions (see the other test fixtures).